tags/not_support.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 🚨 This is not a support channel! 🚨 
+color: orange
+---
+
+<:waaa_prism:1041361839528878160> **Problem in launcher**
+👉 <#1032130809055940690> 👈 
+
+<:fried:1032296608768995368> **Problem with mods**
```

Comment:
```suggestion
<:fried:1032296608768995368> **Problem with using mods in-game**
```

Reply:
That would make it seem like it's not for crashes